test/test_remote_io.py

```diff
@@ -58,7 +60,7 @@ def test_http_reader_iterdatapipe(self):
         self.assertTrue(io.BufferedReader, type(stream))
 
         # __len__ Test: returns the length of source DataPipe
-        self.assertEqual(1, len(http_reader_dp))
+        self.assertEqual(3, len(http_reader_dp))
```

Comment:
```suggestion
        self.assertEqual(1, len(http_reader_dp))
```
After the above change, the value should be kept as `1`